website/content/docs/k8s/installation/deployment-configurations/single-dc-multi-k8s.mdx

```diff
@@ -96,6 +98,10 @@ $ kubectl get secret consul-gossip-encryption-key ${HELM_RELEASE_SERVER}-consul-
 Now we can switch to the second Kubernetes cluster where we will deploy only the Consul clients
```

Comment:
```suggestion
Switch to the second Kubernetes cluster where you will deploy the Consul clients
```
Every instance of "we" should either be "you" or revised into an imperative statement with an implied "you". Per our style guide, use "we" when referring to HashiCorp. We do not use the plural first person (the royal we).

Reply:
Thank you, @trujillo-adam, for the review. I will incorporate the changes you have proposed.